hack/e2e.sh

```diff
@@ -144,16 +143,12 @@ echo "Acquired Workload Cluster Control Plane IP: $WORKLOAD_CONTROL_PLANE_ENDPOI
 
 # Only build and upload the image if we run tests which require it to save some $.
 if [[ -z "${GINKGO_FOCUS+x}" ]]; then
-  # save the docker image locally
-  make e2e-image
+  # Save the docker images locally
+  make e2e-images
   mkdir -p /tmp/images
-  docker save gcr.io/k8s-staging-cluster-api/capv-manager:e2e -o "$DOCKER_IMAGE_TAR"
-
-  # store the image on gcs
-  login
-  E2E_IMAGE_SHA=$(docker inspect --format='{{index .Id}}' gcr.io/k8s-staging-cluster-api/capv-manager:e2e)
-  export E2E_IMAGE_SHA
-  gsutil cp ${DOCKER_IMAGE_TAR} gs://capv-ci/"$E2E_IMAGE_SHA"
+  docker save \
+    "gcr.io/k8s-staging-capi-vsphere/cluster-api-vsphere-controller-${ARCH}:dev" \
```

Comment:
Not sure if the different image name works

Reply:
good catch